lib/models/interaction.js

```diff
@@ -66,6 +66,7 @@ export default (provider) => class Interaction extends hasFormat(provider, 'Inte
       'grantId',
       'lastSubmission',
       'deviceCode',
+      'cid',
```

Comment:
can you add this to https://github.com/panva/node-oidc-provider/blob/v8.x/example/my_adapter.js#L131-L148 too?

Reply:
No problem, let me know if you need the description I provided altered.